ledger/sandbox-classic/BUILD.bazel

```diff
@@ -411,9 +411,6 @@ server_conformance_test(
 
 server_conformance_test(
     name = "conformance-test-exceptions",
```

Comment:
I think we can just drop the whole suite? Exceptions are enabled by default.

Reply:
Good catch.

Reply:
fb4fc905d976543c3b09d5599ae47000c5bb4b12